C-Plus-Plus/ds/Trie_implementation.cpp

```diff
@@ -0,0 +1,159 @@
+//Tries are tree based structures used for efficient retrieval of key form huge set of strings
```

Comment:
@devanshiv123 make all single-line comments int one multiline comment.

Reply:
@iamrajiv I have made the changes. I have removed unnecessary line breaks and uneccessary indents but i havn't removed all indentations as i thought that they were necessary. Please tell if they are also to be removed.